### PR TITLE
telegram-downloader 0.14.1

### PR DESCRIPTION
**Pull request by iyear**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by iyear** [comment]
Hi, do I need to fix "Commit style" failure?

**Comment by chenrui333** [comment]
> Hi, do I need to fix "Commit style" failure?

you need to squash the commits into one with `telegram-downloader 0.14.1` as commit msg

File: Formula/t/telegram-downloader.rb [hunk 1]
```diff
@@ -22,10 +23,13 @@ def install
     ldflags = %W[
       -s -w
       -X github.com/iyear/tdl/pkg/consts.Version=#{version}
-      -X github.com/iyear/tdl/pkg/consts.Commit=#{tap.user}
+      -X github.com/iyear/tdl/pkg/consts.Commit=#{Utils.git_short_head}
```
Comment (chenrui333) [inline Formula/t/telegram-downloader.rb:26]: actually no need to do this.

**Comment by chenrui333** [comment]
@iyear thanks for your first PR to homebrew-core! 🎉

**Comment by iyear** [comment]
Thanks for the review. But I need to write the version and commit info in the CLI to make it easier for users to feedback bugs in issues. What should I do?

**Comment by chenrui333** [comment]
> But I need to write the version and commit info in the CLI to make it easier for users to feedback bugs in issues. What should I do?

why is the version not good enough? the homebrew build is always a clean build from the release tag, and release tag should be one to one mapping to the release commit, isn't that enough?

**Comment by chenrui333** [comment]
no problem, i think commit is good for dirty build :)

**Review by chenrui333 — APPROVED** [review]

**Review by p-linnane — APPROVED** [review]

**Pull request merged**